tests/integration/security/sds_ingress/util/util.go

```diff
@@ -183,6 +183,7 @@ func VisitProductPage(ing ingress.Instance, host string, callType ingress.CallTy
 			Cert:       tlsCtx.Cert,
 			CallType:   callType,
 			Address:    endpointIP,
+			Port:       endpointPort,
```

Comment:
For HTTPAddress we just return `fmt.Sprintf("http://%s:%d", ip, nodePort)`. Why do we need to return IP and Port separately here? It seems they should be the same